src/main/kotlin/net/emdal/tank/Graph.kt

```diff
@@ -41,24 +42,25 @@ data class Graph(val query: String = "", val aliases: List<Pair<String, Entity?>
   ): Graph {
     return this.copy(
       query = query + "-[$alias${block().body()}]->",
-      aliases = aliases + (alias to null)
+      aliases = aliases + (alias to emptyList())
     )
   }
 
   fun <T : Relationship> relationship(
     alias: String = "",
-    madeFrom: T,
+    vararg madeFrom: T,
     block: T.() -> String? = { null }
-  ): Graph = relationship(madeFrom, alias, block)
+  ): Graph = this.copy(
+    query = query + "-[$alias:${madeFrom.map(Relationship::type).joinToString("|")}${madeFrom.first().block().body()}]->",
```

Comment:
This line is getting difficult to read. Could we split some of it into extension functions with describing names maybe?

Reply:
See the previous lines that are similar to this too.